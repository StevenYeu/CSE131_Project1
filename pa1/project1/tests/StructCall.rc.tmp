Error, "project1/tests/StructCall.rc": 
  Number of arguments (2) differs from number of parameters (0).
Error, "project1/tests/StructCall.rc": 
  Number of arguments (0) differs from number of parameters (1).
Error, "project1/tests/StructCall.rc": 
  Number of arguments (0) differs from number of parameters (1).
Error, "project1/tests/StructCall.rc": 
  Illegal call to overloaded function MYS_C.
Compile: failure.
