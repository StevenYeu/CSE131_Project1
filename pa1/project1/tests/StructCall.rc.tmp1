  Number of arguments (2) differs from number of parameters (0).
  Number of arguments (0) differs from number of parameters (1).
  Number of arguments (0) differs from number of parameters (1).
  Illegal call to overloaded function MYS_C.
Compile: failure.
