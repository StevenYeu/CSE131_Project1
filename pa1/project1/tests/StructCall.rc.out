Error, "/tmp/cs131_testrunner_FQ0qMF/file.rc": 
  Number of arguments (2) differs from number of parameters (0).
Error, "/tmp/cs131_testrunner_FQ0qMF/file.rc": 
  Number of arguments (0) differs from number of parameters (1).
Error, "/tmp/cs131_testrunner_FQ0qMF/file.rc": 
  Number of arguments (0) differs from number of parameters (1).
Error, "/tmp/cs131_testrunner_FQ0qMF/file.rc": 
  Illegal call to overloaded function MYS_C.
Compile: failure.
